packages/sensors_plus/example/pubspec.yaml

```diff
@@ -0,0 +1,24 @@
+name: sensors_plus_example
+description: Demonstrates how to use the sensors_plus plugin.
+publish_to: "none"
+
+dependencies:
+  flutter:
+    sdk: flutter
+  sensors_plus: ^1.0.0
+  sensors_plus_tizen:
+    path: ../
+
+dev_dependencies:
+  flutter_driver:
+    sdk: flutter
+  integration_test: ^1.0.2
```

Comment:
Didn't this work?

```suggestion
  integration_test:
    sdk: flutter
```

Reply:
After listening to your explanation offline, I've got to know that what you suggested is a more modern way. thanks!